### PR TITLE
Representers frontend patch

### PR DESCRIPTION
**Pull request by dem4ron**
### This is a collection of minor frontend changes, fixes:
- Feedback editor is anchored to bottom on vertical screen resize
- Handles expansion/compression of editor: 
Compressed by default if no feedback, Expanded by default if feedback. 
On focus expands.
On blur compresses if there is no markdown at all.
- Format big numbers. 1984 --> 1,984
- Loads the already selected feedback_type (minor backend update too)
- MostPopular tag's styling is moved to inline. Line-height is decreased according to the design
- Correct data is shown in PreviewModal's footer
- Line-height adjustment of "We imported your previous feedbacks.."

### CODE REVIEW

File: app/javascript/components/types.ts [hunk 1]
```diff
@@ -441,6 +441,7 @@ export type Representation = {
   track: { title: string; iconUrl: string }
   numSubmissions: number
   feedbackHtml: string
+  feedbackType:RepresentationFeedbackType|""
```
Comment (ErikSchierboom) [inline app/javascript/components/types.ts]: I'll fix this to return `null` if it isn't already set. Makes more sense

Comment (dem4ron) [reply app/javascript/components/types.ts]: ok, yes I have derived this from the current return format

File: app/javascript/components/mentoring/representation/modals/PreviewAutomationModal.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ export function PreviewAutomationModal({
 }: PreviewAutomationModalProps): JSX.Element {
   const [selectedExample, setSelectedExample] = useState<number>(0)
 
+
```
Comment (iHiD) [inline app/javascript/components/mentoring/representation/modals/PreviewAutomationModal.tsx]: I don't think you want this.

File: app/javascript/components/mentoring/representation/right-pane/RepresentationFeedbackEditor.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React, { FocusEventHandler, MouseEventHandler, useCallback, useEffect, useState } from 'react'
+import { MarkdownEditor } from '../../../common'
+import { MarkdownEditorHandle } from '../../../common/MarkdownEditor'
+
```
Comment (iHiD) [inline app/javascript/components/mentoring/representation/right-pane/RepresentationFeedbackEditor.tsx:10]: Double white space

File: tailwind.config.js [hunk 4]
```diff
@@ -253,6 +253,7 @@ module.exports = {
       paragraph: '150%',
       code: '160%',
       huge: '170%',
+      18: '18px',
```
Comment (iHiD) [inline tailwind.config.js]: I'd rather not have this here and stick with percentages as a general

Comment (iHiD) [reply tailwind.config.js]: So inline it if you want to.

**Review by iHiD — APPROVED** [review]

File: app/javascript/components/mentoring/representation/modals/PreviewAutomationModal.tsx (hunk 2 above)
Comment (dem4ron) [reply app/javascript/components/mentoring/representation/modals/PreviewAutomationModal.tsx:31]: you mean the empty line?

File: app/javascript/components/mentoring/representation/right-pane/RepresentationFeedbackEditor.tsx (hunk 3 above)
Comment (dem4ron) [reply app/javascript/components/mentoring/representation/right-pane/RepresentationFeedbackEditor.tsx:10]: cleaning it up

File: tailwind.config.js (hunk 4 above)
Comment (dem4ron) [reply tailwind.config.js:256]: ok

**Comment by ErikSchierboom** [comment]
@dem4ron Let me know when you're done fixing things

**Comment by dem4ron** [comment]
@ErikSchierboom  I'm done fixing things! :)

**Comment by iHiD** [comment]
Commit message: "Make improvements to Representers front-end"

**Pull request merged**